tests/cross-language/go/decrypt_test.go

```diff
@@ -0,0 +1,88 @@
+package cltf
+
+import (
+	"encoding/base64"
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+
+	"github.com/godaddy/asherah/go/appencryption"
+	"github.com/godaddy/asherah/go/appencryption/pkg/crypto/aead"
+	"github.com/godaddy/asherah/go/appencryption/pkg/kms"
+	"github.com/godaddy/asherah/go/appencryption/pkg/persistence"
+	"github.com/pkg/errors"
+)
+
+var (
+	decryptedPayload string
+	encryptedPayload string
+)
+
+func ihaveencryptedDatafrom(filename string) error {
+	data, err := ioutil.ReadFile(fmt.Sprintf("%s%s", FileDirectory, filename))
+	if err != nil {
+		panic(err)
+	}
+
+	encryptedPayload = string(data)
+
+	return nil
+}
+
+func idecrypttheencryptedData() error {
+	crypto := aead.NewAES256GCM()
+	manager, _ := kms.NewStatic(KeyManagementStaticMasterKey, crypto)
+	metastore := persistence.NewSQLMetastore(connection)
+	policy := appencryption.NewCryptoPolicy()
+	config := &appencryption.Config{
+		Service: DefaultServiceID,
+		Product: DefaultProductID,
+		Policy:  policy,
+	}
+
+	factory := appencryption.NewSessionFactory(config, metastore, manager, crypto)
+	defer factory.Close()
+
+	sess, err := factory.GetSession(DefaultPartitionID)
+	if err != nil {
+		panic(err)
```

Comment:
Is there a benefit to using `panic` here (and below) instead of `return err`? Or `godog` convention?

Reply:
returning error now. A lot of this was copy-paste from the referenceapp. Should we remove the `panic` from there as well?